imports/react-ui/settings/knowledgebase/components/topic/topic.js

```diff
@@ -0,0 +1,192 @@
+import React, { PropTypes } from 'react';
+import Select from 'react-select-plus';
+import {
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  Button,
+  ButtonToolbar,
+  Modal,
+} from 'react-bootstrap';
+import ReactMarkdown from 'react-markdown';
+import SelectBrand from '../SelectBrand';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { CommonItem } from '../common';
+
+class KbTopic extends CommonItem {
+  constructor(props, context) {
+    super(props, context);
+
+    let code = '';
+
+    // showed install code automatically in edit mode
+    if (props.item) {
+      code = this.constructor.getInstallCode(props.item._id);
+    }
+
+    this.handleBrandChange = this.handleBrandChange.bind(this);
+
+    this.state = {
+      code,
+      copied: false,
+      selectedCategories: this.getSelectedCategories(),
+    };
+  }
+
+  getSelectedCategories() {
+    const { item } = this.props;
+    return (item && item.categoryIds) || [];
+  }
+
+  handleBrandChange() {
+    if (this.props.item && this.props.item._id) {
+      const code = this.constructor.getInstallCode(this.props.item._id);
+      this.setState({ code, copied: false });
+    }
+  }
+
+  static installCodeIncludeScript(type) {
+    return `
+      (function() {
+        var script = document.createElement('script');
+        script.src = "${Meteor.settings.public.CDN_HOST}/${type}Widget.bundle.js";
```

Comment:
Import Meteor from 'meteor/meteor' module.

Reply:
i am getting an error when i import it.

meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:930 Exception from Tracker recompute function:

TypeError: Cannot read property 'public' of undefined
    at Function.installCodeIncludeScript (topic.js:82)
    at Function.getInstallCode (topic.js:99)
    at new KbTopic (topic.js:33)
    at